packages/orchestrator-ui-components/src/components/WfoTable/WfoBasicTable/WfoBasicTable.tsx

```diff
@@ -49,25 +58,33 @@ export const WfoBasicTable = <T,>({
     onCriteriaChange,
     onUpdateDataSorting,
     onDataSearch,
+    sorting,
 }: WfoBasicTableProps<T>) => {
     const { theme } = useOrchestratorTheme();
-    const { basicTableStyle } = getStyles(theme);
+    const { basicTableStyle, basicTableWithStatusColorColumn } =
+        getStyles(theme);
+    const styles = columns.hasOwnProperty(WFO_STATUS_COLOR_FIELD)
+        ? basicTableWithStatusColorColumn
+        : basicTableStyle;
 
     return (
-        <EuiBasicTable
-            css={basicTableStyle}
-            items={data}
-            columns={mapWfoTableColumnsToEuiColumns(
-                columns,
-                hiddenColumns,
-                dataSorting,
-                onUpdateDataSorting,
-                onDataSearch,
-            )}
-            pagination={pagination}
-            onChange={onCriteriaChange}
-            loading={isLoading}
-        />
+        <EuiFlexItem css={styles}>
```

Comment:
The rows in the service ticket table are too high, might have something to do with the applied style